### ISSUE TITLE
Homekit integration with go2rtc

### ISSUE THREAD

**Issue by latargere** [blakeblackshear/frigate-hass-addons#162]
Hi,

It seems the integration of homekit using the embedded go2rtc doesn't work.
If we use the go2rtc direct add-on, there is no issue.
Could it be because the configuration doesn't set the host_network to true ?

**Comment by NickM-27** [comment]
yes, this is expected and the reason is due to no host network which is on purpose because frigate uses a number of ports internally

**Comment by latargere** [comment]
Thanks for the information so there are only 2 solutions, one is to use an external go2rtc and forward to frigate or use internal go2rtc and use homekit bridge or something similar but it will re encode the stream
